x/protorev/types/keys.go

```diff
@@ -10,3 +10,36 @@ const (
 	// RouterKey defines the module's message routing key
 	RouterKey = ModuleName
 )
+
+const (
+	prefixTokenPairRoutes = iota + 1
```

Comment:
Why do we set this one to iota + 1 and don't set the others? 

Is this standard prefix naming convention?

Reply:
Standard prefix naming convention, it increments it by one for every value that you add after.